src/api/dir.h

```diff
@@ -28,12 +28,12 @@ enum nspire_dir_type {
 
 struct nspire_dir_item {
 	char name[240];
-	unsigned long size, date;
+	uint32_t size, date;
```

Comment:
On IRC, Vogtinator requested size_t. Ditto below.

Reply:
@Vogtinator why do you want `size_t` instead of `uint32_t`, which you say is used internally? It's not like running this on a 64-bit machine will magically make the calculator be able to store files larger than 4GB.